go/cmd/server/logging/filelogstorage_test.go

```diff
@@ -0,0 +1,89 @@
+package logging
+
+import (
+	"path/filepath"
+
+	"context"
+	"reflect"
+	"testing"
+
+	"github.com/apache/arrow/go/v8/arrow"
+	"github.com/apache/arrow/go/v8/arrow/array"
+	"github.com/apache/arrow/go/v8/arrow/memory"
+	"github.com/apache/arrow/go/v8/parquet/file"
+	"github.com/apache/arrow/go/v8/parquet/pqarrow"
+	"github.com/feast-dev/feast/go/internal/test"
+	"github.com/feast-dev/feast/go/protos/feast/types"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFlushToStorage(t *testing.T) {
+	table, err := GenerateLogsAndConvertToArrowTable()
+	defer table.Release()
+	assert.Nil(t, err)
+	offlineStoreConfig := map[string]interface{}{
+		"path": ".",
+	}
+	fileStore, err := NewFileOfflineStore("test", offlineStoreConfig)
+	assert.Nil(t, err)
+	err = fileStore.FlushToStorage(array.Table(table))
+	assert.Nil(t, err)
+	logPath, err := filepath.Abs(filepath.Join(".", "log.parquet"))
+	assert.Nil(t, err)
+	w, err := CreateOrOpenLogFile(logPath)
+	assert.Nil(t, err)
+	ctx := context.Background()
+
+	pf, err := file.NewParquetReader(w)
```

Comment:
`file.OpenParquetFile` does both file opening and create parquet reading
`CreateOrOpenLogFile` should be actually `CreateParquetWriter` function

Reply:
Done

Reply:
Fixed.